fluent-bit-to-loki/cmd/out_loki.go

```diff
@@ -85,14 +85,21 @@ func FLBPluginInit(ctx unsafe.Pointer) int {
 	level.Info(paramLogger).Log("LineFormat", conf.LineFormat)
 	level.Info(paramLogger).Log("DropSingleKey", conf.DropSingleKey)
 	level.Info(paramLogger).Log("LabelMapPath", fmt.Sprintf("%+v", conf.LabelMap))
+	level.Info(paramLogger).Log("ReplaceOutOfOrderTS", fmt.Sprintf("%+v", conf.ReplaceOutOfOrderTS))
 	level.Info(paramLogger).Log("DynamicHostPath", fmt.Sprintf("%+v", conf.DynamicHostPath))
 	level.Info(paramLogger).Log("DynamicHostPrefix", fmt.Sprintf("%+v", conf.DynamicHostPrefix))
-	level.Info(paramLogger).Log("DynamicHostSulfix", fmt.Sprintf("%+v", conf.DynamicHostSulfix))
+	level.Info(paramLogger).Log("DynamicHostSuffix", fmt.Sprintf("%+v", conf.DynamicHostSuffix))
```

Comment:
Add a release note `action operator` that this flag must be adjusted.

Reply:
What flag. The suffix is custom string

Reply:
https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml#L367

Reply:
This is actually a flag with key - value, and the key name is changed in this PR.
So with the new release it will not work with the old configuration and it is correct to add a release note about this